keyboards/daisy/config.h

```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x7169
+#define VENDOR_ID       0x4B50
+#define PRODUCT_ID      0xD7DC
 #define DEVICE_VER      0x0501
 #define MANUFACTURER    KTEC
 #define PRODUCT         Daisy
-#define DESCRIPTION     qmk port for Daisy
+#define DESCRIPTION     QMK + VIA port for Daisy
```

Comment:
```suggestion
```

Reply:
Not sure I understand? Revert this change?

Reply:
I wouldn't need to contain the notion of via, but that doesnt matter since #9732 and the decision to remove.